### PR TITLE
Update screen shot and username path for Textmate 2

### PR DESCRIPTION
**Pull request by marcelinollano**
This PR implements:

- Update screen shot using Texmate 2 and `@2x` resolution.
- For Textmate 2 you need to change the clone path to your own username.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,7 +16,7 @@ style.
 
 ### Textmate 2.x
 
-    git clone https://github.com/pmeinhardt/tm-github-theme.git /Users/pmeinhardt/Library/Application\ Support/TextMate/Managed/Bundles/Themes.tmbundle/Support/web-themes/github
+    git clone https://github.com/pmeinhardt/tm-github-theme.git /Users/<your-username>/Library/Application\ Support/TextMate/Managed/Bundles/Themes.tmbundle/Support/web-themes/github
```
Comment (pmeinhardt) [inline README.md]: I should've thought about this before. We could use `HOME` from the shell environment so it works for all users:

```
$HOME/Library/Application\ Support/TextMate/Managed/Bundles/Themes.tmbundle/Support/web-themes/github
```

Comment (marcelinollano) [reply README.md]: Yup, thats better.

**Comment by pmeinhardt** [comment]
Love the screenshot. That's awesome, thank you 😃

**Comment by marcelinollano** [comment]
Haha no problem, great little theme.

**Pull request merged**